scripts/ci/run-integration-tests

```diff
@@ -140,8 +140,12 @@ function setup {
     echo "==> ERROR: Port ${PROXY_PORT} is already in use"
     exit 1
   fi
-  echo "==> Starting http-server…"
-  http-server -sp ${PROXY_PORT} dist&
+  if [ ! -d ${PROJECT_ROOT}/dist ]; then
+    echo "==> ERROR: no dist folder! forgot npm run build?"
+    exit 1
```

Comment:
I like the decision to not run `npm build` from here. Yeah, could me more convenient, but it could also mask some setup errors we wouldn't spot then